src/medication-request/medication-request.resource.tsx

```diff
@@ -116,10 +122,14 @@ function buildPrescriptionsTableRow(
 
 function computeStatus(orderStatuses: Array<string>) {
   if (orderStatuses.includes("active")) {
-    // if any if active, return active
+    // if any are active, return active
     return "active";
-  } else if (orderStatuses.includes("cancelled")) {
-    // if none are active, then if any are cancelled, return cancelled
+  } else if (orderStatuses.includes("stopped")) {
+    // if none are active, and any are stopped, return expired
```

Comment:
This is tangential to most of this commit, just provides support for a new "stopped" ("expired") status added as part of this ticket:
https://issues.openmrs.org/browse/FM2-480